### ISSUE TITLE
Hosting on Netlify vs GH Pages

### ISSUE THREAD

**Issue by reshamas** [scikit-learn/blog#53]
Initially, I had the Data Umbrella blog hosted on GH Pages. There were some issues with the tags, and since Netlify had more options, the developer who created the DU blog template suggested Netlify.

## Reference
- DU Blog:  https://blog.dataumbrella.org

One of the benefits of Netlify is we can see a preview of the blog before accepting the PR. 
Maybe this is also possible with GH Pages, but it needs to be set up?  

Something we could explore later. 

https://github.com/data-umbrella/data-umbrella.github.io/pull/10#issuecomment-1022314583

<img width="680" alt="Screen Shot 2022-01-26 at 10 40 19 AM" src="https://user-images.githubusercontent.com/2507232/151195422-21a12393-1a08-4923-af21-4d84eaa38943.png">

cc:  @laurburke

**Comment by reshamas** [comment]
@ogrisel @thomasjpfan 
Looks like there is an issue open for this one.

**Comment by reshamas** [comment]
@jjerphan 
If you have some time, perhaps we can implement this? 
Hosting the blog on Netlify instead of GH Pages would make it easier to see the previews.

**Comment by jjerphan** [comment]
Sorry, I hardly have time. :confused:

**Comment by ogrisel** [comment]
Indeed, have previews in PRs would be quite nice. But maybe it's possible to configure a netlify account only to get the previews while keeping the final deployment on github pages?

It might be possible to get the final deployment of https://blog.scikit-learn.org on netflify while keeping the main https://scikit-learn.org website on github pages. It's just that I am not 100% sure about the implications in terms of DNS and TLS certificate configuration.

**Comment by ogrisel** [comment]
In any case, feel free to experiment with netlify and then ping me again when you need me to update the DNS config.
